internal/npm_install/generate_build_file.js

```diff
@@ -124,48 +126,54 @@ function flattenDependencies(pkgs) {
 /**
  * Handles Bazel files in npm distributions.
  */
-function handleBazelFiles(pkg) {
-  if (pkg._previouslyProcessed) {
-    // This npm package has already been processed and bazel files
-    // from the npm distribution prefixed with `_`. However, there are
-    // now generated Bazel files which must be filtered out.
-    pkg._files = pkg._files.filter(file => {
-      const basenameUc = path.basename(file).toUpperCase();
-      if (basenameUc === 'BUILD' || basenameUc === 'BUILD.BAZEL') {
-        return false;
-      }
-      return true;
-    });
-  } else {
-    // This is the first processing of this npm package.
-    // All Bazel files in the npm distribution should be renamed by
-    // adding a `_` prefix so they can be preserved for bazel workspaces
-    // and so that file targets don't cross package boundaries.
-    pkg._files = pkg._files.map(file => {
-      const basename = path.basename(file);
-      const basenameUc = basename.toUpperCase();
-      if (basenameUc === 'WORKSPACE' || basenameUc === 'BUILD' || basenameUc === 'BUILD.BAZEL') {
+function hideBazelFiles(pkg) {
+  pkg._files = pkg._files.map(file => {
+    const basename = path.basename(file);
+    const basenameUc = basename.toUpperCase();
+    if (basenameUc === 'BUILD' || basenameUc === 'BUILD.BAZEL') {
+      if (ERROR_ON_BAZEL_FILES) {
+        console.error(`npm package '${pkg._dir}' has a Bazel BUILD file '${file}'
```

Comment:
Nit: I think this should point to the README for the hide-bazel-files package, which should have the install instructions and the explanation.

Reply:
Good idea

Reply:
Done